backend/src/lib/api-docs/constants.ts

```diff
@@ -596,7 +596,8 @@ export const RAW_SECRETS = {
       "The slug of the project to list secrets from. This parameter is only applicable by machine identities.",
     environment: "The slug of the environment to list secrets from.",
     secretPath: "The secret path to list secrets from.",
-    includeImports: "Weather to include imported secrets or not."
+    includeImports: "Weather to include imported secrets or not.",
+    tagSlugs: "The comma seperated tag slugs to filter secrets"
   },
```

Comment:
I think we should remove tag slugs in the schema and just make tag name be required to be a slug. Users don't yet use slugs in production clients yet since it wasn't supported before so i think we can safely make this change in the backend

Reply:
Removed tag names instead of tag slug